src/spicy/CMakeLists.txt

```diff
@@ -16,14 +16,30 @@ zeek_add_subdir_library(
     BIFS
     spicy.bif)
 
+# Since HILTI does not support preprocessor macro we do the version replacement in CMake.
+#
+# TODO(bbannier): Use a static file again once we drop support for Spicy versions before 1.14.0.
```

Comment:
Do we need the backwards compatibility at all these days? I thought since we moved the Spicy logic code into Zeek proper (vs as an external plugin) we don't really mind anymore if it can still build against a previous, external versions of Spicy.